src/calculateBullsAndCows.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function calculateBullsAndCows(secretNumber, guess) {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < secretNumber.length; i++) {
```

Comment:
Try to avoid using old traditional for loops. Instead, use specific ones like filter/map/foreach/reduce. Here fits reduce.